src/test/java/com/uid2/shared/attest/UidCoreClientTest.java

```diff
@@ -62,18 +62,18 @@ public void Download_Succeed_RequestSentWithExpectedParameters() throws IOExcept
 
     @Test
     public void Download_AttestInternalFail_ExceptionThrown() throws IOException, AttestationResponseHandlerException {
-        AttestationResponseHandlerException exception = new AttestationResponseHandlerException(401, "test failure");
+        AttestationResponseHandlerException exception = new AttestationResponseHandlerException(AttestationResponseCode.AttestationFailure, "test failure");
         doThrow(exception).when(mockAttestationResponseHandler).attest();
 
         CloudStorageException result = assertThrows(CloudStorageException.class, () -> {
             uidCoreClient.download("https://download");
         });
-        String expectedExceptionMessage = "download https://download error: http status: 401, test failure";
+        String expectedExceptionMessage = "download error: AttestationResponseCode: AttestationFailure, test failure";
         assertEquals(expectedExceptionMessage, result.getMessage());
     }
 
     @Test
-    public void Download_Attest401_AttestCalledTwice() throws CloudStorageException, IOException, InterruptedException, AttestationResponseHandlerException {
+    public void Download_Attest401_getOptOut_NotCalled() throws CloudStorageException, IOException, AttestationResponseHandlerException {
```

Comment:
What change is this test update related to?

Reply:
I removed the code that calls attest a second time after the first one returns a 401 as that code is not used in production (we log if it is called), and if it returns a 401, calling a second time will get the same response. That test then failed as it checks for 2 calls. Renamed the test as the thing it should be checking is that we don't call getOptOut after getting a 401